sources/osg/RenderStage.js

```diff
@@ -71,6 +71,17 @@ define( [
         getCamera: function () {
             return this.camera;
         },
+        getPositionnedAttribute: function ( node ) {
```

Comment:
Fixed & repushed
@cedricpinson moved it to shadowMap
@jtorresfabra Good spot, fixed to positioned.